predicators/ground_truth_models/kitchen/options.py

```diff
@@ -0,0 +1,109 @@
+"""Ground-truth options for the Kitchen environment."""
+
+from typing import Dict, Sequence, Set
+
+import numpy as np
+from gym.spaces import Box
+
+try:
+    from mujoco_kitchen.utils import \
+        primitive_and_params_to_primitive_action, \
+        primitive_name_to_action_idx
+    _MJKITCHEN_IMPORTED = True
+except ImportError:
+    _MJKITCHEN_IMPORTED = False
+from predicators import utils
+from predicators.ground_truth_models import GroundTruthOptionFactory
+from predicators.ground_truth_models.kitchen.operators import \
+    KitchenGroundTruthOperatorFactory
+from predicators.structs import Action, Array, Object, ParameterizedOption, \
+    ParameterizedPolicy, ParameterizedTerminal, Predicate, State, \
+    STRIPSOperator, Type
+
+
+class KitchenGroundTruthOptionFactory(GroundTruthOptionFactory):
+    """Ground-truth options for the Kitchen environment."""
+
+    @classmethod
+    def get_env_names(cls) -> Set[str]:
+        return {"kitchen"}
+
+    @classmethod
+    def get_options(cls, env_name: str, types: Dict[str, Type],
+                    predicates: Dict[str, Predicate],
+                    action_space: Box) -> Set[ParameterizedOption]:
+        assert _MJKITCHEN_IMPORTED
+        # Operators
+        operators = KitchenGroundTruthOperatorFactory.get_operators(
+            env_name, types, predicates)
+
+        # Reformat names for consistency with other option naming.
+        def _format_name(name: str) -> str:
+            return "".join([n.capitalize() for n in name.split(" ")])
+
+        options: Set[ParameterizedOption] = set()
+        for op in operators:
+            if "MoveTo" in op.name or "Push" in op.name:
+                val = "move_delta_ee_pose"
+            if isinstance(primitive_name_to_action_idx[val], int):
```

Comment:
hold on; if the above `if` statement doesn't fire, then `val` will not be defined right? Can you change around the logic here to fix that? (perhaps just assert `MoveTo` or `Push` are in `op.name`?)